### DISCUSSION TITLE
Add an option to show identity of users who upvoted a discussion

### DISCUSSION THREAD

**Discussion by Stvad** [community/community#3537] [Discussions]
It can be valuable to be able to see who upvoted/dowvoted a given topic for the feature request use case. (For example in a fashion similar to hove identity of people who reacted with an emoji is shown)

That way I as a project maintainer can reach out to specific interested people for early testing/feedback/etc.

**Comment by PranamBhat** [comment]
Hi @Stvad 

You can see that right now.

Can you please elaborate a little more!

**Reply by Stvad** [reply]
How can I do that? When I hover over the upvote counter - nothing happens 🙁 
![image](https://user-images.githubusercontent.com/81347/111249638-4061bd00-85c9-11eb-848c-56333fc2fc1c.png)

**Reply by PranamBhat** [reply]
@Stvad This one only count is available at the moment. It will be a new feature request going forward. 

You can also bring it here : [https://github.community/](https://github.community/)

**Reply by Stvad** [reply]
So just to confirm - I can't see who upvoted tho discussion now, right? 

Also, I thought this was a good place to add feature requests? hence this post 🙂

**Reply by PranamBhat** [reply]
No. Only "**Reactions**" you can see like that. For "**Upvotes**" we can not see who has upvoted while hovering.

Here you  can find if this feature is already under development [https://github.com/github/roadmap/projects/1](https://github.com/github/roadmap/projects/1)

**Reply by janosh** [reply]
This is still not possible it seems?

@PranamBhat Could you link a specific issue?

**Reply by khinshankhan** [reply]
Seems the roadmap was moved to https://github.com/orgs/github/projects/4247/views/1, which is nice because of the searchability. However, there doesn't seem to be anything related to showing upvoters 🤔 

The feature doesn't seem to exist even in the API https://developer.github.com/v4/
<img width="320" alt="Screen Shot 2022-07-19 at 11 24 23 PM" src="https://user-images.githubusercontent.com/22206867/179889722-f03674b4-a23e-44a8-bceb-a6ccb8e51b29.png">

I guess the only workaround would be asking users to leave a +1 reaction or something.

**Reply by d-kleine** [reply]
I think this is about the "arrow up" reaction icon in the "Discussions" section, not the "thumbs up". When hovering over the "arrow up" icon people have given in a discussion, you cannot see their user names.

Would be great if this feature could be implemented / fixed.

**Comment by drout0** [comment]
Could add a checkmark for the user to choose whether he would like to remain anonymous when upvoting.

**Comment by noopurintel** [comment]
+1 @Stvad  (because upvote info is still invisible 🙂)

**Comment by interifter** [comment]
+1
